### PR TITLE
hardhat deploy flow for alt chain

### PR DESCRIPTION
**Pull request by kumaryash90**

### CODE REVIEW

File: packages/cli/src/core/builder/hardhat.ts [hunk 1]
```diff
@@ -87,19 +87,24 @@ export class HardhatBuilder extends BaseBuilder {
...
             !info.abi
           ) {
             logger.debug("Skipping", contractPath, "(no bytecode or metadata)");
             continue;
           }
 
           const bytecode = info.evm.bytecode.object;
-          const deployedBytecode = info.evm.deployedBytecode.object;
-          const { metadata, abi } = info;
+          const deployedBytecode = info.evm.deployedBytecode
```
Comment (joaquim-verges) [inline packages/cli/src/core/builder/hardhat.ts:97]: what's the explanation here? deployedBytecode should always be there no?

Comment (kumaryash90) [reply packages/cli/src/core/builder/hardhat.ts:97]: was trying this out for alt chains which have different structure for their compiled artifacts

Comment (kumaryash90) [reply packages/cli/src/core/builder/hardhat.ts:97]: closing this PR for now

**Pull request closed without merging**